functorch/_src/decompositions.py

```diff
@@ -381,6 +381,43 @@ def native_layer_norm(input: Tensor, normalized_shape: List[int], weight: Option
     return (out, mean, rstd)
 
 
+@register_decomposition(aten.native_layer_norm_backward)
+def native_layer_norm_backward(grad_out: Tensor, input: Tensor, normalized_shape: List[int], mean: Tensor, rstd: Tensor, weight: Optional[Tensor], bias: Optional[Tensor], output_mask: List[bool]) -> Tuple[Tensor, Tensor, Tensor]:
+    input_shape = input.shape
+    input_ndim = input.dim()
+
+    axis = input_ndim - len(normalized_shape)
+    inner_dims = input_shape[axis:]
+    outer_dims = input_shape[:axis]
+    N = prod(inner_dims)
+
+    x_hat = (input - mean) * rstd
+    grad_x_hat = []
+    if weight is not None:
+        grad_x_hat = grad_out * weight
+    else:
+        grad_x_hat = grad_out
+    a = grad_x_hat * N
+    b = aten.sum(grad_x_hat, inner_dims, True)
+    c1 = grad_x_hat * x_hat
+    c2 = aten.sum(c1, inner_dims, True)
+    c3 = x_hat * c2
+
+    inner = (a - b) - c3
+
+    d_weight = []
```

Comment:
You might want to make these something like `aten.new_empty(input, (0,))` since I think compilers like Torchscript will deal better with that than returning a list (which doesn't quite match the original signature I believe?)

Reply:
Ideally we could just return `None`, but that might also cause type signature issues since in C++ this function is expected to return `(Tensor, Tensor, Tensor)`